src/Nethermind/Nethermind.Runner/configs/mainnet.cfg

```diff
@@ -16,8 +16,6 @@
     "PivotHash": "0x24290b7c113efe59095ff1d3afc15efec735655e6d4065d24704759fa5c9a308",
     "PivotTotalDifficulty": "58750003716598352816469",
     "FastBlocks": true,
-    "AncientBodiesBarrier": 11052984,
-    "AncientReceiptsBarrier": 11052984,
```

Comment:
Do we want to remove these values completely? If user decide to enable them he can set value close to the head and CL will not be able to sync deposit contract. Maybe we can introduce something like `Sync.EnableBarrier`?

Reply:
Fair, but I think it's better option to have default value for this. User may decide to set it to current head

Reply:
If that is desired, we can just make another config file, like `mainet-barriers`, for example

Reply:
I'd better go with `mainnet-light` or something but not sure.